desktop/SmesherManager.ts

```diff
@@ -297,8 +297,17 @@ class SmesherManager extends AbstractManager {
     );
 
     await updateSmeshingOpts(this.genesisID, opts);
+  };
+
+  updatePostProvingOpts = async (provingOpts: PostProvingOpts) => {
+    const { nonces, threads } = provingOpts;
 
-    return true;
+    return updateSmeshingOpts(this.genesisID, {
+      'smeshing-proving-opts': {
+        'smeshing-opts-proving-nonces': nonces,
+        'smeshing-opts-proving-threads': threads,
+      },
+    });
```

Comment:
I updated the solution, currently, it is wrapped into handleIPC, to properly wait for the response, and reuse updateSmeshingOpts fn, but not updateSmeshingConfig fn, because if we are doing the run and update threads and nonces ( if not inited or so ), we don't have an update from node-config and other opts are empty and config is rewriting.